build.sbt

```diff
@@ -63,8 +63,6 @@ lazy val model = crossProject(JVMPlatform, JSPlatform)
   .jsSettings(commonModuleTest: _*)
   .jvmSettings(commonJVMSettings)
 
-val curTime = System.currentTimeMillis()
```

Comment:
This was replaced by `buildInfoOptions += BuildInfoOption.BuildTime`

Reply:
We moved it out or resource changes would trigger a recompilation, this shouldn't be an issue with vite anymore

Reply:
You're right, I had forgotten about this.